raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -192,8 +192,10 @@ contract TokenNetwork is Utils {
 
     event ChannelSettled(
         uint256 indexed channel_identifier,
+        address indexed participant1,
```

Comment:
Is there a reason you chose to make those parameters indexed?

Reply:
No, I was replicating the behaviour of the other events - I removed it just now because it is more expensive and not strictly necessary.